packages/plugins/plugin-onetrust/README.md

```diff
@@ -0,0 +1,76 @@
+# @segment/analytics-react-native-plugin-onetrust
+
+Plugin for adding support for [OneTrust](https://onetrust.com/) CMP to your React Native application.
+
+## Installation
+
+You will need to install the `@segment/analytics-react-native-plugin-onetrust` package as a dependency in your project:
+
+Using NPM:
+
+```bash
+npm install --save @segment/analytics-react-native-plugin-onetrust react-native-onetrust-cmp
+```
+
+Using Yarn:
+
+```bash
+yarn add @segment/analytics-react-native-plugin-onetrust react-native-onetrust-cmp
+```
+
+## Usage
+
+Follow the [instructions for adding plugins](https://github.com/segmentio/analytics-react-native#adding-plugins) on the main Analytics client:
+
+After you create your segment client add `OneTrustPlugin` as a plugin, order doesn't matter, this plugin will apply to all device mode destinations you add before and after this plugin is added:
+
+```ts
+import { createClient } from '@segment/analytics-react-native';
+import { OneTrustPlugin } from '@segment/analytics-react-native-plugin-onetrust';
+import OTPublishersNativeSDK from 'react-native-onetrust-cmp';
+
+const segment = createClient({
+  writeKey: 'SEGMENT_KEY',
+});
+
+segment.add({
+  plugin: new OneTrust(OTPublishersNativeSDK, ['C001', 'C002', '...']),
+});
+
+// device mode destinations
+segment.add({ plugin: new BrazePlugin() });
+```
+
+## Support
+
+Please use Github issues, Pull Requests, or feel free to reach out to our [support team](https://segment.com/help/).
+
+## Integrating with Segment
+
+Interested in integrating your service with us? Check out our [Partners page](https://segment.com/partners/) for more details.
+
+## License
+
+```
+MIT License
+
+Copyright (c) 2021 Segment
```

Comment:
Minor: Copyrights should probably be updated to 2023

Reply:
The dangers of copy and paste I see, will fix it